Program.cs

```diff
@@ -721,103 +722,117 @@ public static void Props()
             Pause();
             Console.Clear();
         }
-
-
-        public static void attack()
+            
+        //This is the attack method. I have reworked it so that there is a global player health variable. This means the method can be used again if needed. It also takes input for the computer health now
+        //The idea is at base level it is simply attack and defence with a higher roll needed to hit if you are attacking someone defending. You can also pick up the knife or the crowbar which increase your damage
+        public static void Attack(int compHP)
```

Comment:
it would be cool to have the computer to have a weapon inventory as well that will randomly pick from

Reply:
Decided not to add this as it would add an even larger amount of variance to the fight and make some attempts won or lost based purely on the random weapon generated. The current randomness of the fight can be mostly alleviated by getting a weapon. Also I don't think it would make sense in this case for the NPC to have a weapon as they are not expecting a fight.

Reply:
that is fair. This can be resolved now.